interfaces/builtin/framebuffer.go

```diff
@@ -0,0 +1,111 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const framebufferConnectedPlugAppArmor = `
+# Description: Allow reading and writing to the universal framebuffer (/dev/fb*) which
+# gives privileged access to the console framebuffer.
+
+/dev/fb[0-9]* rw,
+/dev/graphics/fb[0-9]* rw,
```

Comment:
Please remove `/dev/graphics/fb[0-9]* rw,` as it doesn't appear to be used on Linux. If this is in error, please provide documentation for where it is found and update the udev backend (below) as needed.